erpnext/patches/v13_0/create_website_items_imagefix.py

```diff
@@ -0,0 +1,9 @@
+import frappe
+
+def execute():
+	"Copy website_image field from Item if image is not already populated"
+	for website_item_name in frappe.db.get_list('Website Item', {'image': ("=", "")}, pluck='name'):
+		witem = frappe.get_doc('Website Item', website_item_name)
+		image = frappe.db.get_value('Item', witem.item_code, 'website_image')
+		witem.image = image
+		witem.save()
```

Comment:
I think we can do this for all Website Items where the `website_image` in Item does not match that of Website Item.
The update could include a simple db update (query) to map `website_image` and thumbnail (for safety). The save will make the patch very slow.
I'll push this change

Reply:
Do you mean you're going to rewrite it? I agree it could be written more efficiently with a db.set_value.
But like I said, worth checking `home.html` to see which image field it uses (image) and was intended to use and figuring out whether the `website_image` field is required at all? The DocType `Image Field` setting is set to `image`.
While you're looking at the template you'll notice `website_image_alt` is not used.